WalletWasabi.Fluent/Views/Wallets/Settings/WalletCoinJoinSettingsView.axaml

```diff
@@ -28,7 +30,27 @@
         <Button Theme="{StaticResource AccentButton}" Content="Change" Command="{Binding SelectCoinjoinProfileCommand}" DockPanel.Dock="Right" />
         <TextBlock Text="{Binding SelectedCoinjoinProfileName}" VerticalAlignment="Center" MinWidth="120" Margin="10 0 10 0" />
       </DockPanel>
+
+      <DockPanel>
+        <TextBlock Text="Output Wallet" />
+        <PathIcon Data="{StaticResource warning_regular}" Foreground="{DynamicResource WarningMessageForeground}" DockPanel.Dock="Right" Margin="10,0,0,0" Height="20">
+          <ToolTip.Tip>
+            <TextBlock>
+              - Only opened wallets can be selected.<LineBreak /><LineBreak />
```

Comment:
Opened or loaded?

Reply:
When you are on the Wallet Login screen the button says "Open". I guess the user knows that terminology. The wallet needs to be loaded.

Reply:
We agreed IRL to write loaded.